dashboard/app/views/layouts/application.html.haml

```diff
@@ -2,6 +2,7 @@
 -require '../shared/middleware/helpers/experiments'
 %html{dir: locale_dir, lang: locale}
   %head
+    %script{src: webpack_asset_path("js/localization.js"), 'data-ot-ignore' => ''}
```

Comment:
Can you post the size of this bundle?

Reply:
I'm worried about that too. I'm struggling to build an adhoc to measure the production size. I have trouble building minimized versions locally right now. :(

Reply:
![image](https://github.com/user-attachments/assets/e034f0d8-5dc6-413d-b056-4b7d3b656cfb)

it is 6kb, which is smaller than the onetrust payload. that's not bad. it creates the promise to load the localize widget if the experiment is active for the page and the session.

![image](https://github.com/user-attachments/assets/5fb982a9-13e3-47dc-ae7e-0f1d31fe8dd9)

Localize itself is 20 or so kb.

The widget api call is less than 1 kb and the dictionary is currently similar:

![image](https://github.com/user-attachments/assets/4f38d945-0fa8-4f93-bc12-2fdcb1ae3e6a)